public/controllers/aeon_requests_controller.rb

```diff
@@ -157,7 +157,7 @@ def ancestry_titles_for(record)
     when "resource"
       # see public/app/controller/records_controller#resource
       resource = ResourceView.new(record)
-      breadcrumb_title = (resource.finding_aid_status === 'completed' and not resource.finding_aid_title.nil?) ? resource.finding_aid_title : resource.title
+      breadcrumb_title = (resource.finding_aid_status === 'completed' and not resource.title.nil?) ? resource.title : resource.title
```

Comment:
Same as previous comment. This update is equivalent to `breadcrumb_title = node["title"]`.